rbc/rbclib/__init__.py

```diff
@@ -40,7 +40,17 @@
 """
 
 import llvmlite.binding
-from . import _rbclib
+try:
+    from . import _rbclib
+except ImportError as e:
+    # improve the ImportError error message
+    msg = ("cannot import rbc.rbclib._rbclib: this probably indicates "
```

Comment:
Could you incorporate the original error message to the improve message as it may contain hints to the user about what could be wrong? For instance, use
```
msg = f"failed to import _rbclib: {e}\nThis probably indicates ..."
```

Reply:
I tried to do that, but I think that the error message is much more confusing, because it talks about circular imports. E.g, it looks like this:
```
(rbc-dev) qgpu3 rbc $ python -c 'import rbc'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/antocuni/git/Quansight/rbc/rbc/__init__.py", line 3, in <module>
    from .remotejit import RemoteJIT  # noqa: F401
  File "/home/antocuni/git/Quansight/rbc/rbc/remotejit.py", line 11, in <module>
    from . import irtools
  File "/home/antocuni/git/Quansight/rbc/rbc/irtools.py", line 10, in <module>
    from .targetinfo import TargetInfo
  File "/home/antocuni/git/Quansight/rbc/rbc/targetinfo.py", line 6, in <module>
    from . import libfuncs
  File "/home/antocuni/git/Quansight/rbc/rbc/libfuncs.py", line 4, in <module>
    import rbc.rbclib
  File "/home/antocuni/git/Quansight/rbc/rbc/rbclib/__init__.py", line 44, in <module>
    from . import _rbclib
ImportError: cannot import name '_rbclib' from partially initialized module 'rbc.rbclib' (most likely due to a circular import) (/home/antocuni/git/Quansight/rbc/rbc/rbclib/__init__.py)
This probably indicates that rbc has not been built/installed correctly, possibly because cffi was not available at compilation time
```

That's why I choose to do the other way. Anyway, I'm fine with both ways since this will never be seen by anyone hopefully (thanks to the early check in setup.py)

Reply:
fixed by c22d836